eng/linkerTests/linkerCorrectnessTests.targets

```diff
@@ -0,0 +1,113 @@
+<Project>
+  <PropertyGroup>
+    <LinkerTestDir>$(ArtifactsBinDir)linkerTests/</LinkerTestDir>
+    <LinkerTestProjectsDir>$(LinkerTestDir)projects/</LinkerTestProjectsDir>
+    <TestDotNetPath>$(DotNetRoot)dotnet</TestDotNetPath>
+
+    <ProjectTemplate>$(MSBuildThisFileDirectory)project.csproj.template</ProjectTemplate>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TestConsoleAppSourceFiles Include="$(MSBuildProjectDirectory)/*.cs" />
+
+    <TestSupportFiles Include="$(MSBuildThisFileDirectory)SupportFiles/Directory.Build.props">
+      <DestinationFolder>$(LinkerTestDir)</DestinationFolder>
+    </TestSupportFiles>
+    <TestSupportFiles Include="$(MSBuildThisFileDirectory)SupportFiles/Directory.Build.targets">
+      <DestinationFolder>$(LinkerTestDir)</DestinationFolder>
+    </TestSupportFiles>
+  </ItemGroup>
+
+  <Target Name="CreateTestDir"
+          Inputs="@(TestSupportFiles)"
+          Outputs="@(TestSupportFiles->'%(DestinationFolder)\%(FileName)%(Extension)')">
+    <MakeDir Directories="%(TestSupportFiles.DestinationFolder)" />
+    <Copy SourceFiles="@(TestSupportFiles)" DestinationFolder="%(TestSupportFiles.DestinationFolder)" />
+  </Target>
+
+  <Target Name="GetTestConsoleApps">
+    <ItemGroup>
+      <TestConsoleAppSourceFiles>
+        <ProjectDir>$(LinkerTestProjectsDir)$(MSBuildProjectName)/%(Filename)/</ProjectDir>
+      </TestConsoleAppSourceFiles>
+      <TestConsoleAppSourceFiles>
+        <ProjectFile>%(ProjectDir)project.csproj</ProjectFile>
+        <AssetsFile>%(ProjectDir)obj/project.assets.json</AssetsFile>
```

Comment:
Is `AssetsFile` used? I don't see it being used.

Reply:
mm good point, I don't think so, I'll remove it.